variables.tf

```diff
@@ -52,12 +52,6 @@ variable "namespace_name" {
   description = "A name of namespace for creating"
 }
 
-variable "module_depends_on" {
-  default     = []
-  type        = list(any)
-  description = "A list of explicit dependencies"
-}
-
```

Comment:
Tflint warn that this variable is unused. I removed it, also I can comment out it.